scss/style.scss

```diff
@@ -0,0 +1,1292 @@
+
+        &:last-child {
+            margin: 0px;
+        }
+
+        &__img {
+            width: 44px;
+            height: 62px;
+            background-color: #3949ab;
+            padding: 20px 32px;
+            border-radius: 50%;
+            margin-bottom: 40px;
+        }
+
+        &__text {
+            font-family: OpenSans;
+            font-size: 20px;
+            line-height: 28px;
+            letter-spacing: 0.05em;
+            color: #ffffff;
+            text-align: center;
+        }
+    }
+
+    &__data {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+    }
+
+    &__little-fields {
+        display: flex;
+        justify-content: center;
+        margin-bottom: 33px;
+    }
+
+    &__field {
+        width: 430px;
+        height: 84px;
+        border-radius: 10px;
+        font-family: OpenSans;
+        font-size: 20px;
+        letter-spacing: 0.05em;
+        color: #212121;
+        outline: none;
+        padding-left: 20px;
+    }
+
+    &__big-field {
+        width: 882px;
+        height: 300px;
+        border-radius: 10px;
+        font-family: OpenSans;
+        font-size: 20px;
+        letter-spacing: 0.05em;
+        color: #212121;
+        padding-left: 20px;
+        margin-bottom: 39px;
+        padding-top: 30px;
+    }
+
+    &__button {
+        display: flex;
+        justify-content: center;
+        width: 100%;       
+    }
+
+    &__button a {
+        text-decoration: none;
+        font-family: Roboto;
+        font-size: 20px;
+        font-weight: 700;
+        letter-spacing: 0.05em;
+        color: #212121;
+        background-color: #ffffff;
+        border-radius: 10px;
+        padding: 25px 82px;
+    }
+}
+
+.company {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+
+    &__text {
+        font-family: Roboto;
+        font-size: 20px;
+        line-height: 22px;
+        font-weight: 700;
+        letter-spacing: 0.05em;
+        color: #212121;
+    }
+}
+
+@media (min-width: 1020px) { 
+
+    .latest-works {
+
+        &__buttons--for-phones {
+            display: none;
+        }
+    }
+}
+
+// ТАБЛЕТКА
+@media (min-width: 768px) and (max-width: 979px) { 
+    .container {
+        margin-left: 3%;
+        margin-right: 3%;
+        width: 94%;
+    }
+
+    .main {
+
+        &__name {
+            margin: 50px 0 50px 0;
+        }
+    }
+
+    .header {
+
+        &__top{
+
+            &__menu a {
+                letter-spacing: 0em;
+                margin-right: 7px;
+            }
+        }
+
+        &__content {
+
+            &__title {
+                font-size: 49px;
+            }
+
+            &__button {
+                font-size: 18px;
+            }
+        }
+    }
+
+    .about-us {
+        height: 550px;
+    }
+
+    .our-services {
+        height: 1248px;
+    }
+
+    .latest-works {
+        height: 962px;
+
+        &__buttons--for-phones {
```

Comment:
You need to use more understandable names for modificators  (in this case, `&__buttons--hidden` is more convenient)

Reply:
done